Modules/Components/itkImageRegistrationMethodv4/include/selxItkImageRegistrationMethodv4Component.h

```diff
@@ -131,6 +146,10 @@ class ItkImageRegistrationMethodv4Component :
   std::string m_NumberOfLevelsLastSetBy;
   typename TransformParametersAdaptorsContainerInterfaceType::Pointer m_TransformAdaptorsContainerInterface;
 
+  ComponentBase::ParameterValueType m_RescaleIntensity;
+  bool m_InvertIntensity;
+  float m_MetricSamplingPercentage;
```

Comment:
Is there a specific reason to use 32-bit `float`, instead of 64-bit `double`? In C++ code, `double` is often used by default, for floating points.

Reply:
no particular reason, no, good catch